inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/plugin/core_integration.cpp

```diff
@@ -99,4 +99,8 @@ INSTANTIATE_TEST_SUITE_P(
 INSTANTIATE_TEST_SUITE_P(
         smoke_IEClassLoadNetworkTest, IEClassLoadNetworkTest,
         ::testing::Values("CPU"));
-} // namespace
\ No newline at end of file
+
+// IE Class Load network to default device
+INSTANTIATE_TEST_SUITE_P(smoke_AutoDefaultLoadTests, IEClassLoadNetworkToDefaultPluginTest,
```

Comment:
could you help to change the name to "smoke_Auto_DefaultLoadTests", thank you very much

Reply:
updated, thanks for reminder